Tests/Unit/Controller/Backend/ApplicationControllerTest.php

```diff
@@ -165,7 +165,7 @@ public function listsMyApplications()
         ];
         $GLOBALS['BE_USER'] = $user->reveal();
 
-        $this->jobRepository->findAll()->shouldBeCalled();
+        $this->jobRepository->findByBackendUser($user)->shouldBeCalled();
```

Comment:
This should be `$user->reveal()` instead, otherwise you expect the method to be called with a `ObjectProphecy`.